radarr.json

```
@@ -4,6 +4,7 @@
     "artifact": "https://github.com/freenas/iocage-plugin-radarr.git",
     "official": false,
     "properties": {
+		"allow_mlock": "1"
```

Spacing looks off, and you forgot the comma `,` at the end :wink: 
```suggestion
        "allow_mlock": "1",
```

---

that's what I get for using the github desktop app. thanks for spotting that :)

---

> github desktop app


Not sure how you feel about VS Code, but next time you can give this magic a try.

Open the repo in your web browser, then press the period key `.`